Examples/PythonToInflux/client.py

```diff
@@ -0,0 +1,70 @@
+import argparse
+from influxdb import InfluxDBClient
+import datetime
+
+def main(host='localhost', port=8086):
+    """Instantiate a connection to the InfluxDB."""
+    user = 'root'
+    password = 'root'
+    dbname = 'example'
+    dbuser = 'user'
+    dbuser_password = 'my_secret_password'
+    query = 'select value from cpu_load_short;'
+    json_body = [
+        {
+            "measurement": "cpu_load_short",
+            "tags": {
+                "host": "server01",
+                "region": "us-west"
+            },
+            "time": datetime.datetime.now(),
+            "fields": {
+                "Float_value": 0.64,
+                "Int_value": 3,
+                "String_value": "Text",
+                "Bool_value": True
+            }
+        }
+    ]
+
+    client = InfluxDBClient(host, port, user, password, dbname)
+
+    print("Create database: " + dbname)
+    client.create_database(dbname)
+
+    print("Create a retention policy")
+    client.create_retention_policy('awesome_policy', '3d', 3, default=True)
```

Comment:
Should probably use a real policy? See [documentation](https://docs.influxdata.com/influxdb/v1.7/query_language/database_management/#retention-policy-management)